### PR TITLE
Fix typo in instruction for Kubernetes repository configuration on CentOS/RHEL/Fedora

### PR DESCRIPTION
**Pull request by steve-hardman**
Was trying new release and looking at the instructions https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/change-package-repository/ there is minor correction needed in the instruction involving yum.

Edit: This will fix https://github.com/kubernetes/website/issues/44437

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ version.
    baseurl=https://pkgs.k8s.io/core:/stable:/v{{< param "version" >}}/rpm/
```
Comment (dipesh-rawat) [inline content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md:192]: This one also needs removing.
```suggestion
   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
```

**Comment by dipesh-rawat** [comment]
Looks like this will fix recently reported issue https://github.com/kubernetes/website/issues/44437
_(Would be nice to include it in the [PR description](https://github.com/kubernetes/website/pull/44439#issue-2049612312))_

File: content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md (hunk 1 above)
Comment (steve-hardman) [reply content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md:192]: oops! Forgot this. THanks for review. Updating now.

**Comment by steve-hardman** [comment]
Done updated the PR with issue number.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for the fix!
/lgtm

[Preview Page](https://deploy-preview-44439--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/change-package-repository/#switching-to-another-kubernetes-package-repository)

**Comment by dipesh-rawat** [comment]
/retitle Fix typo in instruction for Kubernetes repository configuration on CentOS/RHEL/Fedora

**Comment by steve-hardman** [comment]
> This pull-request has been approved by:
Once this PR has been reviewed and has the lgtm label, please assign [sftim](https://github.com/sftim) for approval. For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).
The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes%2Fwebsite).

Trying to do bot's suggestion. (sorry if it not right thing to do)

/assign @sftim

**Comment by prasad-shirodkar** [comment]
Please fix `content/zh-cn/docs/tasks/administer-cluster/kubeadm/change-package-repository.md` too. I reported #44437.

**Comment by reylejano** [comment]
Confirmed that this PR fixes the repeated "v" in the step for "Change the version in these URLs to the next available minor release, for example:"

/approve

**Pull request merged**